src/core/layout/qgslayoutitemmap.h

```diff
@@ -622,6 +622,9 @@ class CORE_EXPORT QgsLayoutItemMap : public QgsLayoutItem
 
     void mapThemeChanged( const QString &theme );
 
+    //! Applies current visibility presets to the renamed map theme
```

Comment:
since 3.14

Reply:
Actually, I can't really follow the meaning of this docstring.

Reply:
Hi @m-kuhn 
Sorry I only noticed your reply few days ago.
Indeed, now that i reread... From memories, this ensures that if a theme is being used and is renamed, it will not lose the index in the combobox index and break rendering. Would this be better? 

> Renames the active map theme called \a theme to \a newTheme.

The same could be added to the others (2D and 3D map views).
Do we add "since ..." in private slots?

Reply:
Yes, this doc string sounds perfect.

No, `since` is only for public members (and protected or virtual private, but that doesn't apply here)